data/len_histogram.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+# Print a histogram of line lengths for a text file
+awk '{ print length($0) }' $1 | sort | uniq -c | sort -n -r
```

Comment:
It's better to write `"$1"` instead of `$1`.

Right now, if you run
  `len_histogram.sh 'file with spaces.txt'`
it will pass `file`, `with` and `spaces.txt` as separate arguments to awk.